### PR TITLE
London Class 8 - Senami Ahifon - HTML/CSS Week 2

### PR DESCRIPTION
**Pull request by SenamiAhifon**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Senami Ahifon
- Your City: London
- Your Slack Name: Senami A.

# Homework Details

- Module: HTML/CSS
- Week: 2

# Notes

- What did you find easy?

- What did you find hard? Still need more practice on flexboxes

- What do you still not understand?

- Any other notes?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,19 +1,81 @@
...
-</body>
+        <!--end div text-->
+        <button class="button">Learn More</button>
+      </div>
+      <!--end div hero-->
+    </div>
+    <!--end div content-->
+    <p class="main_paragraph">Everyone needs a little Karma.</p>
+
+    <container class="container">
```
Comment (Simon2828) [inline index.html:44]:
```suggestion
    <section class="container">
```

File: index.html [hunk 2]
```diff
@@ -1,19 +1,81 @@
...
+      <div class="img">
```
Comment (Simon2828) [inline index.html:45]: You could make this element a flex container as well as the container class on line 44 to centre the image and the text below it. 
Try this css: 
```
    display: flex;
    justify-content: center;
    align-items: center;
    flex-direction: column;
```

File: index.html [hunk 3]
```diff
@@ -1,19 +1,81 @@
...
+        <img src="img/icon-devices.svg" alt="Devices" />
+        <p class="container_paragraph">Internet for all devices</p>
+      </div>
+
+      <!--end div img-->
+      <div class="img">
```
Comment (Simon2828) [inline index.html:51]: Try to make this `<div>` a flex container as well

**Review by Simon2828 — COMMENTED** [review]
Great stuff Senami! 😁

File: store.html [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+      </text_store> <!--end div main store-->
+
+<store_image class= "store_image">
+<img src= "img/store-image.jpeg" alt="Lady sitting at her desk, holding a coffee mug" style= "height: 189%">  
+<!--image by neel andrew unsplash-->
+</store_image>
+      </div> <!--end div main store-->
+<form>
+  <fieldset>
```
Comment (Simon2828) [inline store.html:45]: Nice use of semantic HTML 👍

File: store.html [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+  <label for= "form_lname" class= "last_name">Last Name*</label>
+  <input type= "text" name="lname" id= "form_name" required>
+  <br>
+  <br>
+</div>
+
+  <label for= "form_address">Address*</label> <br>
+  <input type= "text" name= "address" id= "form_address" class= "address" required>
+ <br>
+ <br>
```
Comment (Simon2828) [inline store.html:59]: Try to use CSS styling rather than `<br>` tags. Flexbox or CSS-grid could help with styling these `<inputs>`

**Review by Simon2828 — COMMENTED** [review]
Nice work, Senami! We'll take another look at CSS-grid tomorrow which will help with the layout alignment 😁

File: index.html [hunk 6]
```diff
@@ -1,19 +1,81 @@
...
+        <img src="img/icon-coffee.svg" alt="steamy cup of hot drink" />
+        <p class="container_paragraph">Boost your productivity</p>
+      </div>
+
+      <!--end div img-->
+      <div class="img">
```
Comment (TimeaReich) [inline index.html:55]: Good job with the website, as suggested above try making this 
     ` <div class="img"> `a flex container as well.

File: index.html [hunk 7]
```diff
@@ -1,19 +1,81 @@
...
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <body>
+    <img src="img/karma-logo.svg" alt="karma-logo" class="logo" />
+    <div class="content">
+      <div class="navbar">
```
Comment (TimeaReich) [inline index.html:22]: Try using the `<nav>` tag here, instead of the `<div>`. It improves the website's semantics.

**Review by TimeaReich — COMMENTED** [review]
Hope the suggestions help to improve this and your next projects. Good job with this Karma page.

**Pull request closed without merging**